gcloud-java-contrib/gcloud-java-nio-examples/src/main/java/com/google/gcloud/nio/examples/ListFilesystems.java

```diff
@@ -43,4 +43,5 @@ private static void listFilesystems() {
       System.out.println("  " + p.getScheme());
     }
   }
+
 }
```

Comment:
You want me to add a blank line after the final closing bracket? The same line that @mziccard asked me to remove [just a little bit ago](https://github.com/GoogleCloudPlatform/gcloud-java/pull/839#discussion_r58479804)?